_posts/2016-12-22-Rust-1.14.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: post
+title: "Announcing Rust 1.14.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.14.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.14.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.14.0][notes] on GitHub.
+1230 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1140-2016-12-22
+
+### What's in 1.14.0 stable
+
+One of the biggest features in Rust 1.14 isn't actually in Rust 1.14: the
+[rustup tool has reached a 1.0 release][rustup], and is now the recomended way
+to install Rust from the project directly. Rustup does a bit more than just
+install Rust:
+
+> rustup installs The Rust Programming Language from the official release
+> channels, enabling you to easily switch between stable, beta, and nightly
+> compilers and keep them updated. It makes cross-compiling simpler with binary
+> builds of the standard library for common platforms. And it runs on all
+> platforms Rust supports, including Windows.
+
+[rustup]: https://internals.rust-lang.org/t/beta-testing-rustup-rs/3316/203
+
+We had [a previous post about Rustup][prev] back in May. You can learn more
+about it there, or by checking it out [on
+GitHub](https://github.com/rust-lang-nursery/rustup.rs)
+
+[prev]: https://blog.rust-lang.org/2016/05/13/rustup.html
+
+Another exciting feature is [experimental support for WebAssembly][wasm] as a
+target, `wasm32-unknown-emscripten`. It is still early days, and there's a lot
+of bugs to shake out, so please give it a try and report them! To give you a
+small taste of how it works, one you have [emscripten] installed, compiling
+some Rust code to WebAssembly is as easy as:
+
+```bash
+$ rustup target add wasm32-unknown-emscripten
+$ echo 'fn main() { println!("Hello, Emscripten!"); }' > hello.rs
+$ rustc --target=wasm32-unknown-emscripten hello.rs
+$ node hello.js
+```
+
+[wasm]: https://users.rust-lang.org/t/compiling-to-the-web-with-rust-and-emscripten/7627
+[emscripten]: http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html
+
+The community has been doing intertesting, experimental work in this area: see
+[Jan-Erik's slides] for the workshop he ran at [Rust Belt Rust] for some
+examples, or check out [Tim's example of the classic TodoMVC project][todomvc].
+This implementation builds off of his [webplatform
+crate](https://crates.io/crates/webplatform), which exposes the DOM to Rust.
+
+[Jan-Erik's slides]: http://www.hellorust.com/emscripten/
+[Rust Belt Rust]: http://www.rust-belt-rust.com/sessions/
+[todomvc]: http://timryan.org/rust-todomvc/
+
+Speaking of platforms, a large number of platforms have gained additional
```

Comment:
If we're talking about experimental support for new platforms, how about mentioning [xargo](https://github.com/japaric/xargo)?